test/fixtures/incoming_test_emails/outlook/incoming_outlook_email.eml

```diff
@@ -0,0 +1,107 @@
+Delivered-To: railsprojects2018@gmail.com
```

Comment:
So this file should be a complete reply email generated by Outlook, which you can use to base the tests around. if you replace this content with an email generated by Outlook in reply to another email, we can restructure the code around parsing an Outlook-style email so that the "quoted text" does not appear in comments. Does that make sense?